src/Components/App/App.js

```diff
@@ -30,8 +35,37 @@ function App() {
 
   return (
     <div className="App">
-      {!dashboardView && < LandingPage grabUserLocationData={grabUserLocationData} grabInputLocationData={grabInputLocationData}/>}
-      {dashboardView && < Dashboard location={location} />}
+      <Switch>
```

Comment:
This routing is looking good! Maybe we can figure out how to get this dashboard view to show up without conditional rendering, but it does work and that's awesome!